models/SDSC_ISG-TrendModel/main.R

```diff
@@ -0,0 +1,15 @@
+# Pull SDSC_ISG-TrendModel from repo
+
+forecast_date <- lubridate::floor_date(lubridate::today(),
+                               unit = "week",
+                               week_start = 7)
+
+filename <- paste0(forecast_date, "-SDSC_ISG-TrendModel.csv")
+
+url <- paste0("https://raw.githubusercontent.com/TaoSunVoyage/covid19-forecast-hub-europe/main/data-processed/SDSC_ISG-TrendModel/", filename)
+
+dir <- here::here("models", "SDSC_ISG-TrendModel", "data-processed")
```

Comment:
```suggestion
dir <- here::here("models", "SDSC_ISG-TrendModel", "data-processed", "SDSC_ISG-TrendModel")
```

Not sure. My recollection about the hub structure is already pretty blurry. But it seems to match what we're doing in other scripts.